Content.Server/Atmos/Components/GridAtmosphereComponent.cs

```diff
@@ -61,6 +61,12 @@ public sealed partial class GridAtmosphereComponent : Component
         [ViewVariables]
         public int HighPressureDeltaCount => HighPressureDelta.Count;
 
+        [ViewVariables]
+        public readonly HashSet<Entity<DeltaPressureComponent>> DeltaPressureEntities = new();
+
+        [ViewVariables]
+        public readonly Dictionary<Vector2i, float> DeltaPressureCoords = new(1000);
```

Comment:
This could use a better field name. Also it'd be nice if you put docs on all the fields you added since they are public.